Binary files cpa2002v009c.orig/fmg and cpa2002v009c/fmg differ
Binary files cpa2002v009c.orig/gpd and cpa2002v009c/gpd differ
diff -urN cpa2002v009c.orig/makefile cpa2002v009c/makefile
--- cpa2002v009c.orig/makefile	2012-10-06 18:06:07.000000000 +0900
+++ cpa2002v009c/makefile	2013-07-24 22:20:13.000000000 +0900
@@ -12,6 +12,8 @@
 libs =
 
 program = specx
+utility = fmg
+prefix=/usr/local
 objs = \
 source/$(program).o \
 source/spmain.o \
@@ -206,10 +208,23 @@
 source/xtoken.o \
 source/zroots.o
 
+default: $(program) $(utility) gpd
 $(program): $(objs)
-	@$(fort) -o $(program) $(flag) $(objs) $(libs)
+	$(fort) -o $(program) $(flag) $(objs) $(libs)
+$(utility): util/fmg.f
+	$(fort) -o $(utility) $(flag) util/fmg.f $(libs)
 .f.o:
 	$(fort) $(flag) -o $(<:.f=.o) -c $<
 
 gpd: source/gpd.f
 	$(fort) $(flag) -o gpd source/gpd.f
+
+install: $(program) $(utility) gpd
+	mkdir -p $(prefix)/bin $(prefix)/share/machikaneyama/in $(prefix)/share/machikaneyama/util
+	cp $(program) $(utility) gpd $(prefix)/bin
+	chmod 755 $(prefix)/bin/$(program) $(prefix)/share/machikaneyama/in $(prefix)/share/machikaneyama/util
+	cp -p in/* $(prefix)/share/machikaneyama/in
+	cp -p util/* $(prefix)/share/machikaneyama/util
+	chmod 644 $(prefix)/share/machikaneyama/in/* $(prefix)/share/machikaneyama/util/*
+clean:
+	rm -f $(program) $(utility) gpd source/*.o
Binary files cpa2002v009c.orig/source/alphac.o and cpa2002v009c/source/alphac.o differ
Binary files cpa2002v009c.orig/source/asymbl.o and cpa2002v009c/source/asymbl.o differ
Binary files cpa2002v009c.orig/source/atmcor.o and cpa2002v009c/source/atmcor.o differ
Binary files cpa2002v009c.orig/source/atmicv.o and cpa2002v009c/source/atmicv.o differ
Binary files cpa2002v009c.orig/source/atmmas.o and cpa2002v009c/source/atmmas.o differ
Binary files cpa2002v009c.orig/source/atmnum.o and cpa2002v009c/source/atmnum.o differ
Binary files cpa2002v009c.orig/source/atmrot.o and cpa2002v009c/source/atmrot.o differ
Binary files cpa2002v009c.orig/source/aveari.o and cpa2002v009c/source/aveari.o differ
Binary files cpa2002v009c.orig/source/avearr.o and cpa2002v009c/source/avearr.o differ
Binary files cpa2002v009c.orig/source/banden.o and cpa2002v009c/source/banden.o differ
Binary files cpa2002v009c.orig/source/biomix.o and cpa2002v009c/source/biomix.o differ
Binary files cpa2002v009c.orig/source/bravai.o and cpa2002v009c/source/bravai.o differ
Binary files cpa2002v009c.orig/source/brintg.o and cpa2002v009c/source/brintg.o differ
Binary files cpa2002v009c.orig/source/brvsym.o and cpa2002v009c/source/brvsym.o differ
Binary files cpa2002v009c.orig/source/bzmesh.o and cpa2002v009c/source/bzmesh.o differ
Binary files cpa2002v009c.orig/source/bzmrot.o and cpa2002v009c/source/bzmrot.o differ
Binary files cpa2002v009c.orig/source/bzmsmb.o and cpa2002v009c/source/bzmsmb.o differ
Binary files cpa2002v009c.orig/source/cemesh.o and cpa2002v009c/source/cemesh.o differ
Binary files cpa2002v009c.orig/source/cemesr.o and cpa2002v009c/source/cemesr.o differ
Binary files cpa2002v009c.orig/source/cg.o and cpa2002v009c/source/cg.o differ
Binary files cpa2002v009c.orig/source/cgc.o and cpa2002v009c/source/cgc.o differ
Binary files cpa2002v009c.orig/source/cgntcd.o and cpa2002v009c/source/cgntcd.o differ
Binary files cpa2002v009c.orig/source/cgntcs.o and cpa2002v009c/source/cgntcs.o differ
Binary files cpa2002v009c.orig/source/cgnwt.o and cpa2002v009c/source/cgnwt.o differ
Binary files cpa2002v009c.orig/source/cgtabl.o and cpa2002v009c/source/cgtabl.o differ
Binary files cpa2002v009c.orig/source/chebpc.o and cpa2002v009c/source/chebpc.o differ
Binary files cpa2002v009c.orig/source/chklat.o and cpa2002v009c/source/chklat.o differ
Binary files cpa2002v009c.orig/source/chktyp.o and cpa2002v009c/source/chktyp.o differ
Binary files cpa2002v009c.orig/source/chleng.o and cpa2002v009c/source/chleng.o differ
Binary files cpa2002v009c.orig/source/chrasa.o and cpa2002v009c/source/chrasa.o differ
Binary files cpa2002v009c.orig/source/chrdnc.o and cpa2002v009c/source/chrdnc.o differ
Binary files cpa2002v009c.orig/source/cinvrn.o and cpa2002v009c/source/cinvrn.o differ
Binary files cpa2002v009c.orig/source/cinvrx.o and cpa2002v009c/source/cinvrx.o differ
Binary files cpa2002v009c.orig/source/ckmesh.o and cpa2002v009c/source/ckmesh.o differ
Binary files cpa2002v009c.orig/source/clrarc.o and cpa2002v009c/source/clrarc.o differ
Binary files cpa2002v009c.orig/source/clrari.o and cpa2002v009c/source/clrari.o differ
Binary files cpa2002v009c.orig/source/clrarr.o and cpa2002v009c/source/clrarr.o differ
Binary files cpa2002v009c.orig/source/cnsole.o and cpa2002v009c/source/cnsole.o differ
Binary files cpa2002v009c.orig/source/corada.o and cpa2002v009c/source/corada.o differ
Binary files cpa2002v009c.orig/source/corcnf.o and cpa2002v009c/source/corcnf.o differ
Binary files cpa2002v009c.orig/source/corgga.o and cpa2002v009c/source/corgga.o differ
Binary files cpa2002v009c.orig/source/corlsd.o and cpa2002v009c/source/corlsd.o differ
Binary files cpa2002v009c.orig/source/cpaitr.o and cpa2002v009c/source/cpaitr.o differ
Binary files cpa2002v009c.orig/source/cpolin.o and cpa2002v009c/source/cpolin.o differ
Binary files cpa2002v009c.orig/source/cprinv.o and cpa2002v009c/source/cprinv.o differ
Binary files cpa2002v009c.orig/source/cpshft.o and cpa2002v009c/source/cpshft.o differ
Binary files cpa2002v009c.orig/source/crtint.o and cpa2002v009c/source/crtint.o differ
Binary files cpa2002v009c.orig/source/cryrho.o and cpa2002v009c/source/cryrho.o differ
Binary files cpa2002v009c.orig/source/cstatc.o and cpa2002v009c/source/cstatc.o differ
Binary files cpa2002v009c.orig/source/cstate.o and cpa2002v009c/source/cstate.o differ
Binary files cpa2002v009c.orig/source/cubmtr.o and cpa2002v009c/source/cubmtr.o differ
Binary files cpa2002v009c.orig/source/dawson.o and cpa2002v009c/source/dawson.o differ
Binary files cpa2002v009c.orig/source/diffn.o and cpa2002v009c/source/diffn.o differ
Binary files cpa2002v009c.orig/source/dplini.o and cpa2002v009c/source/dplini.o differ
Binary files cpa2002v009c.orig/source/drivtv.o and cpa2002v009c/source/drivtv.o differ
Binary files cpa2002v009c.orig/source/drvlud.o and cpa2002v009c/source/drvlud.o differ
Binary files cpa2002v009c.orig/source/drvmsh.o and cpa2002v009c/source/drvmsh.o differ
Binary files cpa2002v009c.orig/source/dsenum.o and cpa2002v009c/source/dsenum.o differ
Binary files cpa2002v009c.orig/source/dspcpa.o and cpa2002v009c/source/dspcpa.o differ
Binary files cpa2002v009c.orig/source/dspgm.o and cpa2002v009c/source/dspgm.o differ
Binary files cpa2002v009c.orig/source/equarc.o and cpa2002v009c/source/equarc.o differ
Binary files cpa2002v009c.orig/source/equari.o and cpa2002v009c/source/equari.o differ
Binary files cpa2002v009c.orig/source/equarr.o and cpa2002v009c/source/equarr.o differ
Binary files cpa2002v009c.orig/source/eqvlat.o and cpa2002v009c/source/eqvlat.o differ
Binary files cpa2002v009c.orig/source/eranlb.o and cpa2002v009c/source/eranlb.o differ
Binary files cpa2002v009c.orig/source/erfc.o and cpa2002v009c/source/erfc.o differ
Binary files cpa2002v009c.orig/source/erranc.o and cpa2002v009c/source/erranc.o differ
Binary files cpa2002v009c.orig/source/errtrp.o and cpa2002v009c/source/errtrp.o differ
Binary files cpa2002v009c.orig/source/etaopt.o and cpa2002v009c/source/etaopt.o differ
Binary files cpa2002v009c.orig/source/excev.o and cpa2002v009c/source/excev.o differ
Binary files cpa2002v009c.orig/source/excg91.o and cpa2002v009c/source/excg91.o differ
Binary files cpa2002v009c.orig/source/excgga.o and cpa2002v009c/source/excgga.o differ
Binary files cpa2002v009c.orig/source/exclmm.o and cpa2002v009c/source/exclmm.o differ
Binary files cpa2002v009c.orig/source/excmjw.o and cpa2002v009c/source/excmjw.o differ
Binary files cpa2002v009c.orig/source/excpym.o and cpa2002v009c/source/excpym.o differ
Binary files cpa2002v009c.orig/source/excpyv.o and cpa2002v009c/source/excpyv.o differ
Binary files cpa2002v009c.orig/source/excvbh.o and cpa2002v009c/source/excvbh.o differ
Binary files cpa2002v009c.orig/source/excvwn.o and cpa2002v009c/source/excvwn.o differ
Binary files cpa2002v009c.orig/source/exgga.o and cpa2002v009c/source/exgga.o differ
Binary files cpa2002v009c.orig/source/extorg.o and cpa2002v009c/source/extorg.o differ
Binary files cpa2002v009c.orig/source/fczero.o and cpa2002v009c/source/fczero.o differ
Binary files cpa2002v009c.orig/source/fd2ini.o and cpa2002v009c/source/fd2ini.o differ
Binary files cpa2002v009c.orig/source/fd3.o and cpa2002v009c/source/fd3.o differ
Binary files cpa2002v009c.orig/source/finitn.o and cpa2002v009c/source/finitn.o differ
Binary files cpa2002v009c.orig/source/fintgr.o and cpa2002v009c/source/fintgr.o differ
Binary files cpa2002v009c.orig/source/fldf.o and cpa2002v009c/source/fldf.o differ
Binary files cpa2002v009c.orig/source/fxspin.o and cpa2002v009c/source/fxspin.o differ
Binary files cpa2002v009c.orig/source/gcor.o and cpa2002v009c/source/gcor.o differ
Binary files cpa2002v009c.orig/source/gengpt.o and cpa2002v009c/source/gengpt.o differ
Binary files cpa2002v009c.orig/source/genrpt.o and cpa2002v009c/source/genrpt.o differ
Binary files cpa2002v009c.orig/source/gensm.o and cpa2002v009c/source/gensm.o differ
Binary files cpa2002v009c.orig/source/getdtb.o and cpa2002v009c/source/getdtb.o differ
Binary files cpa2002v009c.orig/source/getfil.o and cpa2002v009c/source/getfil.o differ
Binary files cpa2002v009c.orig/source/getnum.o and cpa2002v009c/source/getnum.o differ
Binary files cpa2002v009c.orig/source/getorg.o and cpa2002v009c/source/getorg.o differ
Binary files cpa2002v009c.orig/source/getrtm.o and cpa2002v009c/source/getrtm.o differ
Binary files cpa2002v009c.orig/source/gettmp.o and cpa2002v009c/source/gettmp.o differ
Binary files cpa2002v009c.orig/source/gmtrns.o and cpa2002v009c/source/gmtrns.o differ
Binary files cpa2002v009c.orig/source/gnpset.o and cpa2002v009c/source/gnpset.o differ
Binary files cpa2002v009c.orig/source/gntcds.o and cpa2002v009c/source/gntcds.o differ
Binary files cpa2002v009c.orig/source/gntcs.o and cpa2002v009c/source/gntcs.o differ
Binary files cpa2002v009c.orig/source/gpsort.o and cpa2002v009c/source/gpsort.o differ
Binary files cpa2002v009c.orig/source/green.o and cpa2002v009c/source/green.o differ
Binary files cpa2002v009c.orig/source/gsdatp.o and cpa2002v009c/source/gsdatp.o differ
Binary files cpa2002v009c.orig/source/gtchst.o and cpa2002v009c/source/gtchst.o differ
Binary files cpa2002v009c.orig/source/guesse.o and cpa2002v009c/source/guesse.o differ
Binary files cpa2002v009c.orig/source/guessz.o and cpa2002v009c/source/guessz.o differ
Binary files cpa2002v009c.orig/source/hexmtr.o and cpa2002v009c/source/hexmtr.o differ
Binary files cpa2002v009c.orig/source/hypera.o and cpa2002v009c/source/hypera.o differ
Binary files cpa2002v009c.orig/source/ibrava.o and cpa2002v009c/source/ibrava.o differ
Binary files cpa2002v009c.orig/source/ifkey.o and cpa2002v009c/source/ifkey.o differ
Binary files cpa2002v009c.orig/source/inijip.o and cpa2002v009c/source/inijip.o differ
Binary files cpa2002v009c.orig/source/inqbrv.o and cpa2002v009c/source/inqbrv.o differ
Binary files cpa2002v009c.orig/source/julday.o and cpa2002v009c/source/julday.o differ
Binary files cpa2002v009c.orig/source/kkrsed.o and cpa2002v009c/source/kkrsed.o differ
Binary files cpa2002v009c.orig/source/laguer.o and cpa2002v009c/source/laguer.o differ
Binary files cpa2002v009c.orig/source/lftfil.o and cpa2002v009c/source/lftfil.o differ
Binary files cpa2002v009c.orig/source/lindx.o and cpa2002v009c/source/lindx.o differ
Binary files cpa2002v009c.orig/source/lsqftr.o and cpa2002v009c/source/lsqftr.o differ
Binary files cpa2002v009c.orig/source/lubksb.o and cpa2002v009c/source/lubksb.o differ
Binary files cpa2002v009c.orig/source/ludcmp.o and cpa2002v009c/source/ludcmp.o differ
Binary files cpa2002v009c.orig/source/lwcase.o and cpa2002v009c/source/lwcase.o differ
Binary files cpa2002v009c.orig/source/madlng.o and cpa2002v009c/source/madlng.o differ
Binary files cpa2002v009c.orig/source/mdbsl.o and cpa2002v009c/source/mdbsl.o differ
Binary files cpa2002v009c.orig/source/midexp.o and cpa2002v009c/source/midexp.o differ
Binary files cpa2002v009c.orig/source/mkemap.o and cpa2002v009c/source/mkemap.o differ
Binary files cpa2002v009c.orig/source/mseque.o and cpa2002v009c/source/mseque.o differ
Binary files cpa2002v009c.orig/source/mshatm.o and cpa2002v009c/source/mshatm.o differ
Binary files cpa2002v009c.orig/source/nesbet.o and cpa2002v009c/source/nesbet.o differ
Binary files cpa2002v009c.orig/source/neutrl.o and cpa2002v009c/source/neutrl.o differ
Binary files cpa2002v009c.orig/source/nfqlty.o and cpa2002v009c/source/nfqlty.o differ
Binary files cpa2002v009c.orig/source/phasea.o and cpa2002v009c/source/phasea.o differ
Binary files cpa2002v009c.orig/source/phaseb.o and cpa2002v009c/source/phaseb.o differ
Binary files cpa2002v009c.orig/source/poisna.o and cpa2002v009c/source/poisna.o differ
Binary files cpa2002v009c.orig/source/poisnb.o and cpa2002v009c/source/poisnb.o differ
Binary files cpa2002v009c.orig/source/polin0.o and cpa2002v009c/source/polin0.o differ
Binary files cpa2002v009c.orig/source/polint.o and cpa2002v009c/source/polint.o differ
Binary files cpa2002v009c.orig/source/potenv.o and cpa2002v009c/source/potenv.o differ
Binary files cpa2002v009c.orig/source/prdmtc.o and cpa2002v009c/source/prdmtc.o differ
Binary files cpa2002v009c.orig/source/prdmtr.o and cpa2002v009c/source/prdmtr.o differ
Binary files cpa2002v009c.orig/source/prdrtm.o and cpa2002v009c/source/prdrtm.o differ
Binary files cpa2002v009c.orig/source/prmvec.o and cpa2002v009c/source/prmvec.o differ
Binary files cpa2002v009c.orig/source/prntcs.o and cpa2002v009c/source/prntcs.o differ
Binary files cpa2002v009c.orig/source/prntpn.o and cpa2002v009c/source/prntpn.o differ
Binary files cpa2002v009c.orig/source/pyexch.o and cpa2002v009c/source/pyexch.o differ
Binary files cpa2002v009c.orig/source/qromo.o and cpa2002v009c/source/qromo.o differ
Binary files cpa2002v009c.orig/source/qvolum.o and cpa2002v009c/source/qvolum.o differ
Binary files cpa2002v009c.orig/source/radial.o and cpa2002v009c/source/radial.o differ
Binary files cpa2002v009c.orig/source/radnrl.o and cpa2002v009c/source/radnrl.o differ
Binary files cpa2002v009c.orig/source/ratin0.o and cpa2002v009c/source/ratin0.o differ
Binary files cpa2002v009c.orig/source/readin.o and cpa2002v009c/source/readin.o differ
Binary files cpa2002v009c.orig/source/realh.o and cpa2002v009c/source/realh.o differ
Binary files cpa2002v009c.orig/source/redata.o and cpa2002v009c/source/redata.o differ
Binary files cpa2002v009c.orig/source/reduce.o and cpa2002v009c/source/reduce.o differ
Binary files cpa2002v009c.orig/source/relred.o and cpa2002v009c/source/relred.o differ
Binary files cpa2002v009c.orig/source/rlubks.o and cpa2002v009c/source/rlubks.o differ
Binary files cpa2002v009c.orig/source/rludcm.o and cpa2002v009c/source/rludcm.o differ
Binary files cpa2002v009c.orig/source/rmesha.o and cpa2002v009c/source/rmesha.o differ
Binary files cpa2002v009c.orig/source/rmserb.o and cpa2002v009c/source/rmserb.o differ
Binary files cpa2002v009c.orig/source/rmserr.o and cpa2002v009c/source/rmserr.o differ
Binary files cpa2002v009c.orig/source/rndkpt.o and cpa2002v009c/source/rndkpt.o differ
Binary files cpa2002v009c.orig/source/rotatm.o and cpa2002v009c/source/rotatm.o differ
Binary files cpa2002v009c.orig/source/rrcnv.o and cpa2002v009c/source/rrcnv.o differ
Binary files cpa2002v009c.orig/source/rsymeq.o and cpa2002v009c/source/rsymeq.o differ
Binary files cpa2002v009c.orig/source/sbrnch.o and cpa2002v009c/source/sbrnch.o differ
Binary files cpa2002v009c.orig/source/sbtime.o and cpa2002v009c/source/sbtime.o differ
Binary files cpa2002v009c.orig/source/setarc.o and cpa2002v009c/source/setarc.o differ
Binary files cpa2002v009c.orig/source/setari.o and cpa2002v009c/source/setari.o differ
Binary files cpa2002v009c.orig/source/setarr.o and cpa2002v009c/source/setarr.o differ
Binary files cpa2002v009c.orig/source/spckkr.o and cpa2002v009c/source/spckkr.o differ
Binary files cpa2002v009c.orig/source/specx.o and cpa2002v009c/source/specx.o differ
Binary files cpa2002v009c.orig/source/spmain.o and cpa2002v009c/source/spmain.o differ
Binary files cpa2002v009c.orig/source/stfact.o and cpa2002v009c/source/stfact.o differ
Binary files cpa2002v009c.orig/source/subscr.o and cpa2002v009c/source/subscr.o differ
Binary files cpa2002v009c.orig/source/swpari.o and cpa2002v009c/source/swpari.o differ
Binary files cpa2002v009c.orig/source/swparr.o and cpa2002v009c/source/swparr.o differ
Binary files cpa2002v009c.orig/source/tchmta.o and cpa2002v009c/source/tchmta.o differ
Binary files cpa2002v009c.orig/source/tchstr.o and cpa2002v009c/source/tchstr.o differ
Binary files cpa2002v009c.orig/source/tfp.o and cpa2002v009c/source/tfp.o differ
Binary files cpa2002v009c.orig/source/totalw.o and cpa2002v009c/source/totalw.o differ
Binary files cpa2002v009c.orig/source/transp.o and cpa2002v009c/source/transp.o differ
Binary files cpa2002v009c.orig/source/ty2ity.o and cpa2002v009c/source/ty2ity.o differ
Binary files cpa2002v009c.orig/source/uclock.o and cpa2002v009c/source/uclock.o differ
Binary files cpa2002v009c.orig/source/udate.o and cpa2002v009c/source/udate.o differ
Binary files cpa2002v009c.orig/source/utimer.o and cpa2002v009c/source/utimer.o differ
Binary files cpa2002v009c.orig/source/uutrns.o and cpa2002v009c/source/uutrns.o differ
Binary files cpa2002v009c.orig/source/uxcor.o and cpa2002v009c/source/uxcor.o differ
Binary files cpa2002v009c.orig/source/viomix.o and cpa2002v009c/source/viomix.o differ
Binary files cpa2002v009c.orig/source/vrotat.o and cpa2002v009c/source/vrotat.o differ
Binary files cpa2002v009c.orig/source/vwncor.o and cpa2002v009c/source/vwncor.o differ
Binary files cpa2002v009c.orig/source/xtoken.o and cpa2002v009c/source/xtoken.o differ
Binary files cpa2002v009c.orig/source/zroots.o and cpa2002v009c/source/zroots.o differ
Binary files cpa2002v009c.orig/specx and cpa2002v009c/specx differ
